core/data-resolver/CartService.ts

```diff
@@ -0,0 +1,153 @@
+import { DataResolver } from './types/DataResolver'
+import Task from '@vue-storefront/core/lib/sync/types/Task'
+import CartItem from '@vue-storefront/core/modules/cart/types/CartItem'
+import { TaskQueue } from '@vue-storefront/core/lib/sync'
+import { processLocalizedURLAddress } from '@vue-storefront/core/helpers'
+import config from 'config';
+
+const setServerShippingInfo = async (addressInformation: any): Promise<Task> =>
+  TaskQueue.execute({
+    url: processLocalizedURLAddress(config.cart.shippinginfo_endpoint),
+    payload: {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      mode: 'cors',
+      body: JSON.stringify({ addressInformation })
+    },
+    silent: true
+  });
+
+const getTotals = async (): Promise<Task> =>
+  TaskQueue.execute({
+    url: processLocalizedURLAddress(config.cart.totals_endpoint),
+    payload: {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+      mode: 'cors'
+    },
+    silent: true
+  });
+
+const connectCart = async (guestCart: boolean = false, forceClientState: boolean = false): Promise<Task> => {
```

Comment:
can we check to `getCartToken`? but only in data-resolver, action should be kept as `connect`